app/src/main/java/com/example/momolabfe/ui/stats/StatsFragment.kt

```
@@ -0,0 +1,313 @@
+        val points = stats.points
+        val dateFormatter = DateTimeFormatter.ofPattern("MM/dd")
+
+        val labels = points.map { p ->
+            p.recordDate.format(dateFormatter)
+        }
+
+        val validWeights = points.mapNotNull { it.weight }
+        if (validWeights.isNotEmpty()) {
+            val avgWeight = validWeights.average().toFloat()
+            binding.weeklyAvgWeightContentTv.text =
+                String.format("체중: %.1fkg", avgWeight)
```

detekt (1.23.8)

[warning] String.format("체중: %.1fkg", avgWeight) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)